LISA/LISAJsonRpc.cpp

```diff
@@ -203,25 +204,91 @@ namespace Plugin {
         // and GetStorageDetails have the same 3 input params and only GetStorageDetailsParamsInfo
```

Comment:
We could use:
using GetMetadataParamInfo = GetStorageDetailsParamInfo;
it would not less confusing and easier to change should the any of the structs change.

Reply:
Nice. Done